feat(js/replay): Opt-in options for unmask and unblock (#9422)
getsentry/sentry-docs: platform-includes/replay/privacy-configuration/javascript.mdx

});
```

> Since v8, the options `unblock` and `unmask` do not add default DOM selectors anymore. If you want to keep the default behavior of previous versions, then we recommend you to explicitly specify them in your configuration:

Reviewer: ```suggestion
Starting with v8, the options `unblock` and `unmask` do not add default DOM selectors anymore. If you want to keep the default behavior of previous versions, then we recommend you to explicitly specify them in your configuration:
```

maybe fits better?

> Starting with v8, the options `unblock` and `unmask` do not add default DOM selectors anymore. If you want to keep the default behavior of previous versions, then we recommend you to explicitly specify them in your configuration:

Reviewer 2: ```suggestion
Starting with v8, the options `unblock` and `unmask` do not add default DOM selectors anymore. If you want to keep the default behavior of previous versions, then you should explicitly specify them in your configuration:
```
